servicetalk-encoding-api-internal/src/main/java/io/servicetalk/encoding/api/internal/HeaderUtils.java

```diff
@@ -43,13 +43,13 @@ private HeaderUtils() {
      * <p>
      * If no supported encodings are configured then the result is always {@code null}
      * If no accepted encodings are present in the request then the result is always {@code null}
-     * In all other cases, the first matching encoding (that is NOT {@link ContentCodings#identity()}) is preferred,
```

Comment:
Worth deprecating ContentCodings#identity() ?

Reply:
The class `ContentCodings` is deprecated.

Reply:
does this PR remove the last usage of the class?

Reply:
yes, there were a few places that weren't caught in the previous refactoring. I fixed them here.